### ISSUE TITLE
[Feature] adjust the width of the drop-down list of published topics

### ISSUE THREAD

**Issue by puhitaku** [emqx/MQTTX#979]
#### Motivation

Thank you guys for this great software,

The items in the topic history drop-down list gets truncated when the topic is long (around 20 chars) and sometimes these items lacks the latter part so I can't find the exact topic I want. And unfortunately, topics in this kind of length can occur anytime.

Following is the screenshot:

![image](https://user-images.githubusercontent.com/2425178/177590141-c14736c9-0ba1-44eb-a20b-a6e110ce25ff.png)

#### Detailed design

Horizontal auto-growing the list is the most favorable way I can think of because increasing it with fixed amount will not solve the problem especially when the topic is super long.

#### Alternatives

No alternatives at present.

#### More detail (optional)

N/A

**Comment by ysfscream** [comment]
Hi @puhitaku sorry, I'm late replying, I was on vacation last week and will consider this optimization. But here I would recommend using topic alias for overly long topics, not the alias in mqtt5, you can set it when you add a subscription.

<img width="557" alt="image" src="https://user-images.githubusercontent.com/21299158/178176929-b7a52fbb-d1f0-499a-a793-4cc3c914d35c.png">

**Comment by puhitaku** [comment]
No problems, thanks for a reaction.

Following is the screenshot when I subscribed to `testtopic/#` with alias `test` and published a test payload to `testtopic/#`. The subscription list shows the alias `test` but any other panes including the publish history list don't shorten the topic with the alias.

Alias feature fits nicely in the subscription list but unfortunately it does not solve the issue I posted 😢 

![image](https://user-images.githubusercontent.com/2425178/178761145-36f31383-cb3e-41b5-88c6-0b371d357a0c.png)
